x/ccv/provider/keeper/hooks_test.go

```diff
@@ -33,7 +33,10 @@ func TestValidatorConsensusKeyInUse(t *testing.T) {
 			setup: func(ctx sdk.Context, k providerkeeper.Keeper) {
```

Comment:
This test was hard to follow in some cases but I did not change which validator addrs are input to the methods